.ci/static-checks.sh

```diff
@@ -315,7 +381,14 @@ check_docs()
 
 	for url in $urls
 	do
-		# Ignore the install guide urls that contain a shell variable
+		if [ "$master_branch" != "true" ]
+		then
+			# If the URL is new on this PR, it cannot be checked.
+			echo "$new_urls"|tr ' ' '\n'|grep -q "^${url}$" && \
```

Comment:
/me blinks at the magic a few times. Maybe a short explanatory comment on how this works ;-)

Reply:
I've simplified the code as we don't really need to loop anyhow.